src/azure-cli/azure/cli/__main__.py

```diff
@@ -71,3 +69,7 @@ def cli_main(cli, args):
                     invoke_finish_time - init_finish_time)
     except NameError:
         pass
+
+    telemetry.set_init_time_elapsed("{:.6f}".format(init_finish_time - start_time))
+    telemetry.set_invoke_time_elapsed("{:.6f}".format(invoke_finish_time - init_finish_time))
```

Comment:
Is it better if we only record the `init_finish_time` to telemetry?

Now that we alreay have `start_time`, `init_finish_time` and `end_time`, it seems not necessary here to introduce anthor `invoke_finish_time`.

Reply:
For convenience and consistency purpose, `start_time`, `init_finish_time`, `invoke_finish_time` are all generated with `timeit.default_timer()` while the `start_time` and `end_time` in telemetry session use `datetime.datetime.utcnow()`.